### ISSUE TITLE
'Animated' Warnings

### ISSUE THREAD

**Issue by JorgeOrozcoP** [alexbrillant/react-native-deck-swiper#292]
I have a running deck swipper on my application, but whenever I manually move the cards, I get the following two warnings: 

`Animated: 'useNativeDriver' was not specified. This is a required option and must be explicitly set to 'true' or 'false' 
`

and

` Animated.event now requires a second argument for options `

I managed to hide these warnings using `console.disableYellowBox = true;` I'm pretty new to RA, and I'm wondering if there is a better solution to this. I am using React Native v6.14.4

**Comment by Remato** [comment]
just search animated.timing on folder ~\node_modules\native-base\dist\src\basic\ and add manually useNativeDriver: true or false

**Comment by MayoudP** [comment]
@Remato Can you be more specific please ? 
Got the same error with RN : 0.62

**Comment by Remato** [comment]
> @Remato Can you be more specific please ?
> Got the same warnings with RN : 0.62
> Not a blocking bug tho

you have to set this manually for the component that is giving this warning, so...

you have a folder node_modules with yours components right?
go to src/node_module/{your_component}

where: your_component is `react_native_deck_swipper`
search for file `Swiper.js` open it and add this line here:
![Screenshot from 2020-04-14 16-18-31](https://user-images.githubusercontent.com/8043534/79264846-98e58300-7e6b-11ea-9023-2bb2ec8646c4.png)
```
Animated.timing(this.state.pan, {
      toValue: {
        x: x * SWIPE_MULTIPLY_FACTOR,
        y: y * SWIPE_MULTIPLY_FACTOR
      },
      duration: this.props.swipeAnimationDuration,
      useNativeDriver: true // Add This line
      ...
```
you can fix this errors manually for all components without this line "useNativeDriver".

**Comment by saltyhoney** [comment]
useNativeDriver: false // Add This line


looks like adding `false` also works.

**Comment by Remato** [comment]
> useNativeDriver: false // Add This line
> 
> looks like adding `false` also works.

yeah, the warning disappear if you determine any value for this variable

```useNativeDriver: true```
or
```useNativeDriver: false```

works fine, but this defines whether the native drive will be used or not, if your application does not use it, it can be set to `false`
🙂

**Comment by adavecohen95** [comment]
Won't this just go back whenever I refresh my dependencies

**Comment by MikeMajara** [comment]
as @adavecohen95 points out, this will come back if library needs to be updated. Ideally this would require a PR. I'd be happy to do one, but I'd need more insight into this library... @alexbrillant ?

@Remato this solution leads to the second problem stated in OP: `Animated.event now requires a second argument for options`, any thoughts?

@ all - Is there a reason not to `useNativeDriver` by default?

**Comment by ipuris** [comment]
Hi, I found this thread while googling the exactly same error I experienced in different react-native code, and I think it can be helpful to solve the second problem:

```
Animated.event now requires a second argument for options
```

I use the following code: 

```
Animated.event([null, { dy: someValue }], { useNativeDriver: true, }),
```

which was previously: 

```
Animated.event([null, { dy: someValue }]), // wrong
```

The second error occurs if `Animated.event` has only one argument as the message says, and the function maybe requires only one argument in previous version, but it may changes. 
I don't know where the right place of source code in this library to fix the problem, but I will send PR if possible. 
Before that, I first wrote this comment because I hope this comment would be helpful for other users.

**Comment by ipuris** [comment]
Oh, I'm late 🤦  I found that the following PR already fixes the problem that I said. 

https://github.com/alexbrillant/react-native-deck-swiper/pull/309/commits/fb9fbd6283f4144889aef8e98f9e765fc74b37de
